### PR TITLE
feat: add nav-bar

### PR DESCRIPTION
**Pull request by Shurtu-gal**
### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [x] Make sure you are making a pull request against the **default branch** (left side). Also you should start *your branch* off *default branch*.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.
- [ ] I have added necessary documentation (if appropriate)

### Description
Please describe your pull request.

❤️Thank you!

### Post merge checklist
- [x] Follow steps from the [guidelines for contributing](../CONTRIBUTING.md) to this repository.
- [x] If you are a new contributor, ping in the thread and one of the maintainers will add you to all-contributors list.

### CODE REVIEW

**Comment by Shurtu-gal** [comment]
@riteshsp2000 @ORKO06 @ratnakar5938 
Can you kindly review the changes?

Thank you

**Review by ratnakar5938 — CHANGES_REQUESTED** [review]

File: src/components/content/content.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      link: '/prizes',
+    },
+    {
+      id: 'prizes',
+      name: 'Prizes',
+      link: '/prizes',
+    },
+    
+  ],
+};
```
[the remark this answers is not in the document]
Comment (Shurtu-gal) [reply src/components/content/content.js:51]: Sure

**Review by ratnakar5938 — APPROVED** [review]

**Comment by ratnakar5938** [comment]
@Shurtu-gal change the file structure to that of 3.0

**Comment by ratnakar5938** [comment]
@Shurtu-gal Why are there 2 types of hamburgermenu in the design 
Screenshot (292)
Screenshot (293)

**Comment by ratnakar5938** [comment]
LGTM 🚀

**Review by padmajabhol — APPROVED** [review]

**Pull request merged**